### DISCUSSION TITLE
Having two relay clients, one with TRACKING ON and one without TRACKING

### DISCUSSION THREAD

**Discussion by dorrogeray** [cachewerk/relay#157] [Ideas]
I am not sure how much effort would it be to make `$relay->setOption(Relay::OPT_USE_CACHE, false)` ensure that such connection will NOT create a connection with TRACKING ON, but it would be a huge help - especially in situations where you have 2 sets of highly distinct keys, one set which changes extremely frequently which you dont want to track, and one which is changed rather infrequently and is thus great candidate for tracking, and you want to put them into two separate redis instances / redis clusters.

The same would apply for `Relay\Cluster` class, where I would like to be able to configure one cluster to use tracking and the other to not use it.

**Comment by tillkruss** [comment]
Added in v0.10.0
